specs/capella/beacon-chain.md

```diff
@@ -489,3 +490,59 @@ def process_bls_to_execution_change(state: BeaconState,
         + address_change.to_execution_address
     )
 ```
+
+## Testing
+
+*Note*: The function `initialize_beacon_state_from_eth1` is modified for pure Capella testing only.
+Modifications include:
+1. Use `CAPELLA_FORK_VERSION` as the current fork version.
```

Comment:
```suggestion
1. Use `CAPELLA_FORK_VERSION` as the previous fork version.
```

should this say `previous` as the modification?

Reply:
Ah, I think it changes both. nice catch!